src/SearchIndex/util.ts

```diff
@@ -32,6 +31,7 @@ interface FacetRes {
   [key: string]: FacetRes | string | number | undefined;
 }
 
+// TODO: update this to work with new facet structure (children and options)
 export function formatFacetMetaResponse(facetAggRes: FacetAggRes, taxonomyTrie: FacetDisplayNames) {
   const res: {
     count: number;
```

Comment:
Nit: Should we comment out this function just to make sure no one tries to use it while it is not in sync with the new schema?

Reply:
in retrospect, i should have made this an additive PR but this is currently being used on front end for showing search results. can release together with follow up [PR](https://github.com/mongodb/docs-search-transport/pull/77) which fixes this route again

Reply:
ah fascinating, ok ok didn't realize!